examples/tts/conf/fastpitch_align_finetuning.yaml

```diff
@@ -0,0 +1,182 @@
+name: "FastPitch"
```

Comment:
What is the difference between this file and `fastpitch_align_44100.yaml`?

Reply:
The optimizer specification. We use a simple fixed lr optimizer for the finetuning. Added a comment about it.

Reply:
Would it be easier to just ask users to change via the command line? like `fastpitch.py ~model.optim.sched`. Could be `-model.optim.sched`, don't quite remember hydra specs.

Reply:
Done